.github/ISSUE_TEMPLATE/new_feature

```diff
@@ -0,0 +1,13 @@
+## Desired Behavior
+
+Describe desired behavior for improvements/enhancements.
+What is the problem we want to solve?
+What is the plan?
```

Comment:
nit: wrapping the explanatory text in `<!--` HTML comments `-->` will save anyone opening a new issue the time for deleting said text making it visible only in editing mode

Reply:
Good point, done.